presto-main/src/test/java/com/facebook/presto/sql/planner/TestPlanMatchingFramework.java

```diff
@@ -69,6 +73,24 @@ public void testOutputSameColumnMultipleTimesWithOtherOutputs()
     }
 
     @Test
+    public void testStrictOutput()
+    {
+        assertMinimallyOptimizedPlan("SELECT orderkey, extendedprice FROM lineitem",
+                strictOutput(ImmutableList.of("ORDERKEY", "EXTENDEDPRICE"),
+                        tableScan("lineitem", ImmutableMap.of("ORDERKEY", "orderkey",
```

Comment:
That should work. As long as you aren't introducing any new symbols in the project node, you should be able to use strictOutput with the aliases you already have bound.